### PR TITLE
feat: Add clearAttachments to Scope

### PR DESCRIPTION
**Pull request by xmanu**
## :scroll: Description
We want to expose a method to clear the attachments on the scope.

## :bulb: Motivation and Context
getsentry/sentry-java#1517

## :green_heart: How did you test it?
I added a test to ScopeTest to verify that the `clearAttachments` method really clears all attachments.

## :pencil: Checklist

- [x] I reviewed the submitted code
- [x] I added tests to verify the changes
- [ ] I updated the docs if needed
- [x] No breaking changes

### CODE REVIEW

**Review by marandaneto — APPROVED** [review]

**Comment by marandaneto** [comment]
@xmanu thanks a lot :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * Fix: Handling missing Spring Security on classpath on Java 8 (#1552)
 * Feat: Support transaction waiting for children to finish. (#1535) 
 * Feat: Capture logged marker in log4j2 and logback appenders (#1551)
+* Feat: Allow clearing of attachments in the scope (#1517)
```
Comment (marandaneto) [inline CHANGELOG.md:8]:
```suggestion
* Feat: Allow clearing of attachments in the scope (#1562)
```

Comment (marandaneto) [reply CHANGELOG.md:8]: its the PR id though

**Review by philipphofmann — APPROVED** [review]
Thanks a lot.

**Pull request merged**